doc/man/8/rbd.rst

```diff
@@ -273,14 +273,16 @@ Commands
   (in bytes), the length of the region (in bytes), and either 'zero' or 'data' to indicate
   whether the region is known to be zeros or may contain other data.
 
-:command:`du` [-p | --pool *pool-name*] [*image-spec* | *snap-spec*]
+:command:`du` [-p | --pool *pool-name*] [*image-spec* | *snap-spec*] [--merge]
```

Comment:
i wonder if `--summarize` or `--sum` might be a better name?

Reply:
@tchaikov Thanks for the suggestion, I have push a renamed patch below (I do prefer a full `--summarize` rather than  `--sum`)